README.md

```diff
@@ -51,7 +51,7 @@ With good encryption defaults and anonymous, disposable application services lik
 
 ## Usage
 
-Go to https://godoc.org/github.com/textileio/go-textile-core.
+Go to https://godoc.org/github.com/textileio/go-textile/core.
```

Comment:
Thanks! This should just be "Go to https://godoc.org/github.com/textileio/go-textile"